index.d.ts

```diff
@@ -1,19 +1,28 @@
+declare namespace unusedFilename {
+	interface Options {
+		/**
+		Filename increase function
```

Comment:
This should better explain what it does and some use-cases for it. Also add a code example. Same in the readme.

Reply:
I have added some text, not too many experiences on docs